src/test/java/org/opensearch/ad/MemoryTrackerTests.java

```diff
@@ -76,7 +76,7 @@ public void setUp() throws Exception {
         super.setUp();
         rcfNumFeatures = 1;
         rcfSampleSize = 256;
-        numberOfTrees = 10;
+        numberOfTrees = 30;
```

Comment:
Much needed! An ensemble of 10 trees is quite noisy.

Reply:
the change has already made in previous PRs.  This is just tests :)